### DISCUSSION TITLE
풀리퀘 승인 질문

### DISCUSSION THREAD

**Discussion by glossyyoon** [devrel-kr/HackaLearn#423] [1 Q&A - HackaLearn 운영] [answered]
안녕하세요 어제 개발을 마치고 모든 챌린지를 한꺼번에 완료한 다음 풀리퀘를 보냈는데 오늘 아침에 승인이 안되어있는 것을 확인했습니다.
PR 한 번에 한 챌린지만 올릴 수 있다는 것을 몰랐는데 이미 16일 0시가 지나 다시 풀리퀘를 보낼 수 없는 상황입니다ㅜㅜ 승인이 불가한 것일까요?#384 입니다.

**Comment by justinyoo** [comment, marked as answer]
@glossyyoon 안타깝지만 리뷰어 분들이 계속해서 한번에 PR 하나씩 올려달라는 코멘트를 드렸습니다. 이 부분 해결이 안되어 승인은 났지만, 올려주신 PR 중 하나만 카운트가 되고 나머지는 카운트 되지 않습니다.
